hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommitLocks.java

```diff
@@ -72,10 +77,17 @@ public class TestHiveCommitLocks extends HiveTableBaseTest {
   LockResponse notAcquiredLockResponse = new LockResponse(dummyLockId, LockState.NOT_ACQUIRED);
 
   @BeforeClass
-  public static void initializeSpies() throws Exception {
+  public static void startMetastore() throws Exception {
+    HiveMetastoreTest.hiveConf = new HiveConf();
+    HiveMetastoreTest.hiveConf.set(HiveConf.ConfVars.HIVE_TXN_TIMEOUT.varname, "1s");
+    HiveMetastoreTest.startMetastore();
```

Comment:
Could we create a:
```
HiveMetastoreTest.startMetastore(Map<String, String> override) {
}
```
in the `HiveMetastoreTest` class? It might be more readable than this?